### DISCUSSION TITLE
How to get artist and title info from streaming radio metadata?

### DISCUSSION THREAD

**Discussion by maxborowsky** [DeaDBeeF-Player/deadbeef#2621] [Q&A]
I see the artist and the name of the song in the playlist, i.e. the player correctly receives them when playing the radio. But when I open the properties of the track to copy the artist and the name of the song, the artist is not on the list, and instead of the name of the song - only the name of the radio station.

![Screenshot from 2021-06-19 02-11-05](https://user-images.githubusercontent.com/20723033/122623382-83d63680-d0a4-11eb-8f63-fae0f160d82c.png)
![Screenshot from 2021-06-19 02-11-31](https://user-images.githubusercontent.com/20723033/122623386-88025400-d0a4-11eb-99f9-d506f7cac024.png)

Deadbeef version: 1.8.4 - 1.8.7
OS: Ubuntu 20.04.2 LTS - 21.04, Fedora 33-34

**Comment by kpcee** [comment]
There is a [plugin](https://deadbeef.sourceforge.io/plugins.html) called Copy Info, you can try if it works with it.

**Reply by maxborowsky** [reply]
Yes, Copy Info works, thank you for saying about this plugin. But it seems to me that this solution (sorry!) is a duct tape :-( Why not show this information in Track Properties? As had been done, for example, in AIMP (Windows):
![screenshot_2021_06_19_5](https://user-images.githubusercontent.com/20723033/122653754-7cb83280-d14f-11eb-9ec4-4aaffd711a23.png)
![screenshot_2021_06_19_6](https://user-images.githubusercontent.com/20723033/122653757-8346aa00-d14f-11eb-9462-1f918e1b6be3.png)
